examples/quickstart/jupyter-notebooks/notebooks/03-query/03-approxCountDistinct.ipynb

```diff
@@ -0,0 +1,470 @@
+   "execution_count": null,
+   "id": "f2a19226-6abc-436d-ac3c-9c04d6026707",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import druidapi\n",
+    "import json\n",
+    "import matplotlib\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "\n",
+    "# druid_host is the hostname and port for your Druid deployment. \n",
+    "# In the Docker Compose tutorial environment, this is the Router\n",
+    "# service running at \"http://router:8888\".\n",
+    "# If you are not using the Docker Compose environment, edit the `druid_host`.\n",
+    "\n",
+    "druid_host = \"http://router:8888\"\n",
+    "druid_host\n",
+    "\n",
+    "druid = druidapi.jupyter_client(druid_host)\n",
+    "display = druid.display\n",
+    "sql_client = druid.sql\n",
+    "display.tables('INFORMATION_SCHEMA')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f388633f-195b-4381-98cc-7a2f80f48690",
+   "metadata": {},
+   "source": [
+    "## COUNT(DISTINCT) queries on basic datasets\n",
+    "\n",
+    "Here's a very simple query to find the number of distinct Tail Numbers in the example dataset.\n",
+    "\n",
+    "```sql\n",
+    "SELECT \"Reporting_Airline\", COUNT(DISTINCT \"Tail_Number\") AS \"Events\"\n",
+    "FROM \"On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2005_11\"\n",
+    "GROUP BY 1\n",
+    "ORDER BY 2\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "95a8d8bf-69fa-4266-b171-cb550009e89e",
+   "metadata": {},
+   "source": [
+    "### Running COUNT(DISTINCT) with approximation\n",
+    "\n",
+    "Druid will automatically look for patterns of query that can make use of approximation. In this instance, Druid will identify a match for approximate `COUNT(DISTINCT)`.\n",
+    "\n",
+    "Each data server computes its own intermediate results for merging into the final result set. In a `COUNT(DISTINCT)` query, that means that they will create their own count inside a representation called a [data sketch](https://datasketches.apache.org/). These much smaller objects are then merged together when the query results are finalized, rather than Druid having to combine the individual lists of distinct values from each process.\n",
+    "\n",
+    "This translates into much faster query execution, especially when the intermediate results are large – say when there are a lot of unique values in the source data.\n",
+    "\n",
+    "It also means that the most scalable part of Druid – the individual data servers – do much more of the work, and do so earlier, instead of leaving it to the merge stage.\n",
+    "\n",
+    "> Approximations improve scalability, storage, and memory use - at the cost of some error.\n",
+    "> \n",
+    "> _[Gian Merlino](https://github.com/gianm)_\n",
+    "\n",
+    "Let's run this query with all of Druid's defaults to see what the results are like. (We can safely omit a `__time` filter thanks to the tiny size of the example dataset.)"
```

Comment:
we tend to avoid first person for second person. Maybe "Try running this query... " also avoid parens

Reply:
Removed this from throughout the text.